src/components/Navbar.jsx

```diff
@@ -1,9 +1,15 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faUser } from '@fortawesome/free-solid-svg-icons';
+
 const Navbar = () => (
   <nav className="navbar">
-    <h1>Bookstore</h1>
-    <div className="links-container">
-      <a className="links" href="/">Books</a>
-      <a className="links" href="/categories">Categories</a>
+    <div className="nav-elements">
+      <h1>Bookstore ADE</h1>
+      <div className="links-container">
+        <a className="links BOOKS" href="/">Books</a>
+        <a className="links CATEGORIES" href="/categories">Categories</a>
+      </div>
+      <div className="faUser"><FontAwesomeIcon icon={faUser} /></div>
```

Comment:
- [ ] Nav  h1 still does not look like on  the design, kindly make sure to be in one line
- [ ] Also the vertical line for the **comment** is too close to it, kindly make it as on the  **remove**

Yours: 
![image](https://github.com/adeola003/bookstore/assets/62705947/71f85849-e4b0-4490-b535-f69a4ce1d59a)
Deign:
![image](https://github.com/adeola003/bookstore/assets/62705947/14e011c1-bcea-40ad-923b-f8c6968ee9c7)

Reply:
@GzimAsani the title in the navbar and the from title are not going next line on my side even when I reduce screen up to 700px

Reply:
- Hey @adeola003, ignore this comment I made the check on Firefox and for some reason it was showing like that, just double checked on chrome and it looks fine. I apologize for the confusion